doc/DESIGN-AND-ROADMAP.md

```diff
@@ -61,6 +61,61 @@ Privacy Badger:
 
 #### Further Details
 
+Datastructures:
+
+action_map = { 'google.com': blocked, 'fonts.google.com', 'cookieblocked', 'apis.fonts.google.com': 'user_cookieblock', 'foo.tracker.net': 'allow', 'tracker.net': 'DNT', }
```

Comment:
Looks like the issue is with `'fonts.google.com', 'cookieblocked'`. I submitted a PR to fix this and reformat that section of the docs a bit.